src/SkynetServer/Network/Model/ChannelAction.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SkynetServer.Network.Model
+{
+    internal enum ChannelAction
+    {
+        None,
+        Typing,
+        RecordingAudio
```

Comment:
Should we add something like "taking a picture" here? It annoys me when I ask someone on WhatsApp to send an image of something, then they don't read it and i don't know if they are just in the CameraActivity or ignoring me.